public/Aryan Suri/Sidebar/sidebar.css

```
@@ -536,8 +506,30 @@ a.toggler.off {
 }
 
 @media print {
-  #custom_open, .custom_sidebar {
-    display:none;
+  #custom_open {
+    display: none;
+  }
+  #custom_sidebar {
+    display: none;
+  }
+  #sidebar1 {
+    display: none;
+  }
+  #sidebar2 {
+    display: none;
+  }
+  #sidebar3 {
+    display: none;
+  }
+  #sidebar4 {
+    display: none;
+  }
```

Recommend compress/combining for conciseness. If there is a future purpose to leaving them separate, then feel free to leave as-is.
```suggestion
  #custom_open,
  #custom_sidebar,
  #sidebar1,
  #sidebar2,
  #sidebar3,
  #sidebar4 {
    display: none;
  }
```